### ISSUE TITLE
Solidworks, Autocad and other CAD related softwares

### ISSUE THREAD

**Issue by saqibpervez** [Fmstrat/winapps#271]
Is it possible to run solidworks through rdp like this. On dassaults website, it says solidworks supports vmware and hyper-v. Still i have tried to install solidworks in windows through kvm and it couldn't be activated. Same with virtual box. I don't want to dual boot just because of this. I need solidworks for college project so please dont suggest me alternatives. I dont want to install windows as i hate microsoft from bottom of my heart but if solidworks can't run on linux in any way, i would single boot windows only as my pc doesn't run properly after dual boot.

**Comment by freechelmi** [comment]
Yes it should run in KVM , you shoudl give it a try , the Tutorial is really simple
